.github/workflows/build.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - master
+    - maci-*
```

Comment:
should we remove this?

Reply:
No please... I'd like to have CI running on my branches :)